packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -40,7 +51,7 @@ enum ShortCutLocation {
   Left,
   Right
 }
-
+let currentNode = 0;
```

Comment:
We do not want to keep state on global level as it will often lead to errors when components are reused. This will need to be handled in the react component.